src/engine/Resources/Sound/AudioInstance.ts

```diff
@@ -130,6 +136,12 @@ export class AudioInstance implements Audio {
  * Internal class representing a HTML5 audio instance
  */
 /* istanbul ignore next */
+
+/**
+ * As AudioTagInstance needs to be removed,
```

Comment:
@electro75 In this case I don't think we need to mark as `@obsolete` and can be removed since this is an implementation of `AudioInstance`